lib/travis/shell/generator/bash.rb

```diff
@@ -144,7 +144,7 @@ def handle_else(cmds)
         private
 
           def handle_secure_vars(key, value, options)
-            if options[:echo] && options.delete(:secure)
+            if options[:echo] && options[:secure]
```

Comment:
Since `options` is passed around in this class everywhere (instead of it being an instance variable), and `#handle_secure_vars` is called from `#handle_export`, which then just invokes `#handle_cmd` for the call site to call `export`, I think it is OK to leave the `:secure` key-value pair here.